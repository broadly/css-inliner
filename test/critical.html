<html>
  <head>
    <title>Hello World!</title>
    <style>
      body {
        font:  Helvetica;
      }
      @media screen {
        body {
          font-size: 12pt;
        }
      }
      body, h1 {
        font-weight: bold;
      }
      h1:hover {
        text-decoration: underline;
      }
    </style>
    <link rel="stylesheet" href="extract_rules.css">
    <link rel="stylesheet" href="http://example.com/style.css">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>
