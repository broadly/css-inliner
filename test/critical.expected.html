<html>
  <head><style>body{font:Helvetica}@media screen{body{font-size:12pt}}body,h1{font-weight:bold}h1:hover{text-decoration:underline}body{color:blue}</style>
    <title>Hello World!</title>
    
    
    <link rel="stylesheet" href="http://example.com/style.css">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>
